repo/mk/js.deps.npm.mk

```diff
@@ -29,3 +29,8 @@ deps-npm:
 	if [[ -x node_modules/eslint-config-firecloud/npm-install-peer-dependencies ]]; then \
 		node_modules/eslint-config-firecloud/npm-install-peer-dependencies; \
 	fi
+
+
+.PHONY: deps-npm-prod
+deps-npm-prod:
+	$(NPM) install --prod
```

Comment:
such incomplete docs :) but yes, I would have made the change myself since it's stupid to go with the "shorter" long-option

ps: such bad design to allow for stuff like `-v or --ver or --vsn or --version`, same for `npm i`, same for ...ranting...ranting...ranting :) btw, I don't think `--save-production` works, or at least it didn't work at some point 🙄